content/basics/index.md

```diff
@@ -361,44 +351,11 @@ function myFunction() {
 // this will be hoisted
 var name;
 ```
-... Is valid.
-
-#### Closures
```

Comment:
I don't think this is essential / valuable to know. It's a very contrived example

Reply:
Yep, although is your thought here, don't bother at all with explanation?

Reply:
Yeah I think this is a confusing example for a beginner (and also something that is potentially already learned from other languages, like Python, where you can have nested functions)